packages/twenty-server/src/modules/workflow/workflow-trigger/jobs/workflow-trigger.job.ts

```diff
@@ -68,17 +71,20 @@ export class WorkflowTriggerJob {
         );
       }
 
-      await this.workflowRunnerWorkspaceService.run(
-        data.workspaceId,
-        workflow.lastPublishedVersionId,
-        data.payload,
-        {
+      await this.workflowRunnerWorkspaceService.run({
+        workspaceId: data.workspaceId,
+        workflowVersionId: workflow.lastPublishedVersionId,
+        payload: data.payload,
+        source: {
           source: FieldActorSource.WORKFLOW,
-          name: workflow.name,
+          name:
+            isDefined(workflow.name) && !isEmpty(workflow.name)
+              ? workflow.name
+              : 'Workflow',
```

Comment:
Please create a variable for that
```suggestion
              : DEFAULT_WORKFLOW_NAME,
```